### DISCUSSION TITLE
Universal __mocks__ in workspace

### DISCUSSION THREAD

**Discussion by carlbleick** [vitest-dev/vitest#6130] [Q&A]
I want to create universal mocks for external packages inside my monorepo on root level. I want all packages in the workspace to use the same mocked version of a package.

What configuration is required to achieve that? I need the workspace projects to look for mocks in the `__mocks__` folder in the root of the monorepo.
```
packages/
├─ package-a/
├─ package-b/
__mocks__/
├─ mocked-package
```

I created a minimal stackblitz to show what I am trying to achieve:
https://stackblitz.com/edit/vitest-dev-vitest-kvizof

If this is considered a bad practice then please tell me, otherwise any hint is appreciated! :)

**Comment by sheremet-va** [comment]
I don't think it's possible without creating separate folders in each project re-exporting everything from the root. I think it's a reasonble feature to have, so feel free to open an issue.
